src/Shared/EventArgsFormatting.cs

```diff
@@ -328,9 +330,11 @@ string logOutputProperties
 
             string finalFormat = format.ToString();
 
+            // Reuse the string builder to create the final message
+            StringBuilder formattedMessage = format.Clear();
```

Comment:
Nit: Isn't this not available on some platform?

Reply:
It is nestandard 1.0 so it is everywhere but .NET 3.5-